src/net/fe/overworldStage/fieldskill/Shove.java

```diff
@@ -79,6 +94,22 @@ public static boolean canShove(Grid grid, Unit shover, Unit shovee) {
 		);
 	}
 	
+	public static boolean canShoveWithFog(Grid grid, Unit shover, Unit shovee) {
```

Comment:
I'm not saying we *should* assume this, but if smite is two spaces, and the smallest sight is three spaces, wont a unit always be able to see the space that they'd be shoved into?

Reply:
Right, it's possible to shove enemies. Forgot about that. This is necessary.